github/search.go

```diff
@@ -54,6 +55,36 @@ func (s *SearchService) Repositories(query string, opt *SearchOptions) (*Reposit
 	return result, resp, err
 }
 
+type SingleCommitResult struct {
+	Hash           *string     `json:"hash,omitempty"`
+	Message        *string     `json:"message,omitempty"`
+	AuthorID       int         `json:"author_id,omitempty"`
+	AuthorName     *string     `json:"author_name,omitempty"`
+	AuthorEmail    *string     `json:"author_email,omitempty"`
+	AuthorDate     *string     `json:"author_date,omitempty"`
+	CommitterID    int         `json:"committer_id,omitempty"`
```

Comment:
Any reason these two ints (`AuthorID` and `CommitterID`) aren't pointers, like everywhere else?